rocminfo.cc

```diff
@@ -729,15 +728,15 @@ static void DisplayISAInfo(isa_info_t *isa_i, uint32_t indent) {
   for (int i = 0; i < 3; i++) {
     dim = "Dim[" + std::to_string(i) + "]:";
     printLabelInt(dim.c_str(),
-         reinterpret_cast<uint32_t*>(&isa_i->workgroup_max_dim)[i], indent+1);
+         static_cast<uint32_t>(isa_i->workgroup_max_dim[i]), indent+1);
   }
 
   printLabelInt("Workgroup Max Size:", isa_i->workgroup_max_size, indent);
 
-  printLabel("Grid Max Dimension:", true, indent);
-  printLabelInt("x", isa_i->grid_max_dim.x, indent+1);
-  printLabelInt("y", isa_i->grid_max_dim.y, indent+1);
-  printLabelInt("z", isa_i->grid_max_dim.z, indent+1);
+  printLabel("Grid Max Size per Dimension:", true, indent);
+  printLabelInt("Dim[0]", isa_i->grid_max_dim.x, indent+1);
+  printLabelInt("Dim[1]", isa_i->grid_max_dim.y, indent+1);
```

Comment:
Is there a reason to switch to the Dim[] naming vs. "x, y, z"? The latter was based on HSA docs.

Reply:
see comment for the agent max grid size